migrations/Version20200719160409.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200719160409 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $this->addSql(<<<EOF
```

Comment:
protip, pro ten heredoc lze použít cokoliv - třeba `<<<SQL`, pro který pak funguje správně i syntax highliting a napovídání v IDE

Reply:
to vím, ale mě to ukazuje SQL i s tím EOF